lib/metrics.coffee

```diff
@@ -1,4 +1,3 @@
-crypto = require 'crypto'
```

Comment:
Node built-ins are always cached in memory so I don't think inlining them will ever make a difference.